node/consensus/state/state.go

```diff
@@ -379,12 +422,15 @@ func (ce *ControlEvent) FromBytes(bytes []byte, fragmentFromBytes func([]byte) (
 	case 2:
 		ce.Complaint = &Complaint{}
 		return ce.Complaint.FromBytes(bytes[1:])
+	case 3:
+		ce.ConfigRequest = &ConfigRequest{}
+		return ce.ConfigRequest.FromBytes(bytes[1:])
 	}
 
 	return fmt.Errorf("unknown prefix (%d)", bytes[0])
 }
 
-func (s *State) Process(l types.Logger, ces ...ControlEvent) (*State, []types.BatchAttestationFragment) {
+func (s *State) Process(l types.Logger, ces ...ControlEvent) (*State, []types.BatchAttestationFragment, *ConfigRequest) {
```

Comment:
Will a config request change the state? 
Should a config be also handled by a rule?

Nit - rename s2 into something meaningful... nextState maybe?